src/pages/javascript/tutorials/delete-properties-from-a-object/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Delete Properties from a JavaScript Object
+title: Delete Properties from a Object
```

Comment:
"an Object" too. Sorry, missed this last time.

Reply:
No need for a sorry. Thanks for reviewing. 😄